spec/controllers/admin/blacklisted_email_domains_controller_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'rails_helper'
```

Comment:
I have missed that specs lack `frozen_string_literal` comment. Add them also to specs (though the rule is not enforced well for existing specs…)

Reply:
I also overlooked it.
Thank you for pointing out.
I fixed it earlier.